### PR TITLE
Correct some issues for regression

### PR DESCRIPTION
**Pull request by yonglingchen**

### CODE REVIEW

File: module_utils/stcrest.py [hunk 1]
```diff
@@ -59,12 +59,14 @@ def new_session(self, user_name, session_name, reset_existing=True, kill_existin
...
             rsp = conn.post(url, data=params, timeout=60 * 2)
             log.info("SESSION %s %s -> [%d] %s" % (url, json.dumps(params, indent=4), rsp.status_code, rsp.content))
-
             if rsp.status_code != 409 and rsp.status_code != 200 and rsp.status_code != 201:
                 log.error("Failed to create a session: %d %s" % (rsp.status_code, rsp))
                 return False
 
         self.session = sessionID
+
+        self.conn.headers.update({'Accept': 'application/json', "X-STC-API-Session": self.session})
```
Comment (ronanjs) [inline module_utils/stcrest.py:68]: Is this really needed? The session is already set here:

https://github.com/Spirent/stc-ansible/blob/3d4046d436237a7281c447489e4d7259316013bf/module_utils/stcrest.py#L36

File: playbooks/device-create.yaml [hunk 2]
```diff
@@ -77,7 +77,7 @@
   stc: 
     action: config
     count: 2
-    objects: /EmulatedDevice[@Name='dev-$item']
+    objects: /EmulatedDevice[@Name='client-$item']
```
Comment (ronanjs) [inline playbooks/device-create.yaml:80]: Why is this needed?

File: module_utils/stcrest.py (hunk 1 above)
Comment (yonglingchen) [reply module_utils/stcrest.py:68]: When do reset action, the header will not be updated. It will use the previous session name.

Comment (yonglingchen) [reply module_utils/stcrest.py:68]: > Is this really needed? The session is already set here:
> 
> https://github.com/Spirent/stc-ansible/blob/3d4046d436237a7281c447489e4d7259316013bf/module_utils/stcrest.py#L36

When do reset action, the header will not be updated. It will use the previous session name.

File: playbooks/device-create.yaml (hunk 2 above)
Comment (yonglingchen) [reply playbooks/device-create.yaml:80]: > Why is this needed?

device name is "client-$item" or "server-$item".

Comment (ronanjs) [reply playbooks/device-create.yaml:80]: ok

File: module_utils/stcrest.py (hunk 1 above)
Comment (ronanjs) [reply module_utils/stcrest.py:68]: Ok. Can you then remove then `, "X-STC-API-Session": self.session` from line 36?

Comment (yonglingchen) [reply module_utils/stcrest.py:68]: > Ok. Can you then remove then `, "X-STC-API-Session": self.session` from line 36?

Ok. Let me test it after remove it.

Comment (yonglingchen) [reply module_utils/stcrest.py:67]: > Ok. Can you then remove then `, "X-STC-API-Session": self.session` from line 36?

Done.
